src/main/kotlin/org/rust/lang/core/crate/Crate.kt

```diff
@@ -37,6 +37,8 @@ interface Crate {
     val cfgOptions: CfgOptions
     val features: Map<String, FeatureState>
 
+    val isBuildscriptEvaluated: Boolean
```

Comment:
Let's rename it somehow because the current name is misleading: I suppose it should be false when we don't evaluate build scripts at all but actually, it's true when crate's package doesn't have build script at all.
Something like `canHaveUnknownCfgOptions`